mappings/net/minecraft/client/realms/gui/screen/RealmsInfoScrollableWidget.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_qnehaoor net/minecraft/client/realms/gui/screen/RealmsInfoScrollableWidget
```

Comment:
I don't believe this is realms specific, there's nothing to imply that in the class itself and neither mojang nor yarn use a "realms" name